File: challenges/week5_swappin_aint_easy/solutions/swap/swap.f90
! Fortran
! swap.f90
program swap_example
  implicit none
  integer :: a, b, temp
  a = 1
  b = 2
  print *, "Before: a=", a, " b=", b
  temp = a
  a = b
  b = temp
  print *, "After: a=", a, " b=", b
end program swap_example

File: challenges/week5_swappin_aint_easy/solutions/swap/swap.go
package main

import "fmt"

func swap(a, b int) (int, int) {
    return b, a
}

func main() {
    a, b := 1, 2
    fmt.Println("Before:", a, b)
    a, b = swap(a, b)
    fmt.Println("After:", a, b)
}
